client/my-sites/theme/main.jsx

```diff
@@ -1037,10 +1037,10 @@ class ThemeSheet extends Component {
 		} else if ( isExternallyManagedTheme ) {
 			if ( ! isMarketplaceThemeSubscribed && ! isSiteEligibleForManagedExternalThemes ) {
 				return translate(
-					'Unlock this theme by upgrading to a Business plan and subscribing to this premium theme.'
+					'Unlock this theme by upgrading to a Business plan and subscribing to this theme.'
 				);
 			}
-			return translate( 'Subscribe to this premium theme and unlock all its features.' );
+			return translate( 'Subscribe to this theme and unlock all its features.' );
```

Comment:
With the current code, Line 1043 is likely to be executed even if the user is subscribed to the theme (`isMarketplaceThemeSubscribed = true`) theoretically (though it doesn't seem to be possible as a specification of `isMarketplaceThemeSubscribed = true` AND `isSiteEligibleForManagedExternalThemes = false` does not seem to be ). I suggest the following for reliability and a bit of readability. 

```ts
} else if ( isExternallyManagedTheme && ! isMarketplaceThemeSubscribed ) {
	if ( ! isSiteEligibleForManagedExternalThemes ) {
		return translate(
			'Unlock this theme by upgrading to a Business plan and subscribing to this theme.'
		);
	}
	return translate( 'Subscribe to this theme and unlock all its features.' );
}
```